community/deb.sh

```diff
@@ -79,10 +79,19 @@ detect_os ()
   echo "Detected operating system as $os/$dist."
 }
 
+add_pgdg ()
+{
+  echo "Registering PostgreSQL PGDG dependencies..."
+  echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list
+  apt-get install ca-certificates
+  curl -L https://www.postgresql.org/media/keys/ACCC4CF8.asc 2> /dev/null | apt-key add - &>/dev/null
+}
+
 main ()
 {
   detect_os
   curl_check
+  add_pgdg
```

Comment:
Since I'd like to add a `postgresql-9.5` availability check in here, let's move this after the _Running apt-get update_ phase.